tutorials/034 - Distributed Calls on Remote Cluster.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![AWS SDK for pandas](_static/logo.png \"AWS SDK for pandas\")](https://github.com/aws/aws-sdk-pandas)\n",
+    "\n",
+    "# 7 - Distributed Calls on Remote Cluster\n",
+    "\n",
+    "[SDK for pandas](https://github.com/aws/aws-sdk-pandas) supports distribution of specific calls using [ray](https://www.ray.io/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "pip install awswrangler[\"distributed\"]"
```

Comment:
Can we use `!pip` instead of `pip`. I think it's a convention in notebooks and we use it in other tutorials

Since the distributed version of awswrangler will be in rc at first, we must specify the version for now